code/datums/components/crafting/recipes.dm

```diff
@@ -669,6 +669,14 @@
 	blacklist |= typesof(/obj/item/radio/headset)
 	blacklist |= typesof(/obj/item/radio/intercom)
 
+/datum/crafting_recipe/mothplush
```

Comment:
this crafting recipe is entirely unmentioned in the changelog, also.

Reply:
I would disagree
"Mothroach hide, which can be used to craft a moth plush, and Mothroach meat"
I can be more specific about it, but it's not "entirely unmentioned"

Reply:
Done

Reply:
Done